rust/tools/authorization-logic/src/parsing/AuthLogic.g4

```diff
@@ -60,22 +60,25 @@ assertion
     | fact ':-' flatFact (',' flatFact )* '.' #hornClauseAssertion
     ;
 
+// The IDs following "Export" are path names where JSON files containing
```

Comment:
Can you add a test?

Reply:
There is already a test that checks that policy statements can be properly imported and exported and some tests about the related cryptographic signature checking.

I am guessing that I am over-indexing on the line you are commenting on, and that you want a test of the new functionality from this commit.

Reply:
I meant adding a test where access path has a dot (`.`) in it. I don't think any of the test_inputs meet the criteria?